tradista-gui/src/main/java/finance/tradista/core/daterule/ui/controller/DateRulesController.java

```diff
@@ -154,9 +155,9 @@ public void changed(ObservableValue<? extends Boolean> arg0, Boolean arg1, Boole
 				}
 			}
 		});
-		dateRuleName.setCellValueFactory(new PropertyValueFactory<DateRuleDurationProperty, String>("dateRuleName"));
+		dateRuleName.setCellValueFactory(cellData -> cellData.getValue().getDateRuleName());
 		dateRuleDuration
-				.setCellValueFactory(new PropertyValueFactory<DateRuleDurationProperty, String>("dateRuleDuration"));
+				.setCellValueFactory(cellData -> cellData.getValue().getDateRuleName());
```

Comment:
@bapan25 getDateRuleDuration should be used here instead of getDateRuleName

Reply:
@oasuncion Fixed.